index.html

```diff
@@ -13,35 +13,196 @@
-                <li><a href="#"><img src="media/github.png" alt="github"></a></li>
-                <li><a href="#"><img src="media/Linkedin icon.png" alt="linkedIn"></a></li>
-                <li><a href="#"><img src="media/angel list.png" alt="angel list"></a></li>
-                <li><a href="#"><img src="media/twitter.png" alt="twitter"></a></li>
-                <li><a href="#"><img src="media/medium.png" alt="medium"></a></li>          
+            <ul id="social-items" class="no-style-deco">
+                <li class="social-item"><a href="#"><img src="media/github.png" alt="github"></a></li>
+                <li class="social-item"><a href="#"><img src="media/Linkedin icon.png" alt="linkedIn"></a></li>
+                <li class="social-item"><a href="#"><img src="media/angel list.png" alt="angel list"></a></li>
+                <li class="social-item"><a href="#"><img src="media/twitter.png" alt="twitter"></a></li>
+                <li class="social-item"><a href="#"><img src="media/medium.png" alt="medium"></a></li>          
         </ul>
         </aside>
+        
         <div id="bio">
-        <h1>Hey There. <br> I'm Sylvester</h1>
-        <h2>I'm a Software Developer</h2>
-        <p>I can help you build a product , feature or website. Look through some of 
-            my work and experience! If you like what you see and have a project you need coded, <br>
+        <h1 class="orange-color">Hey There. <br> I'm Sylvester</h1>
+        <h2 class="white">I'm a Software Developer</h2>
+        <p class="white">I can help you build a product , feature or website. Look through some of 
+            my work and experience! If you like what you see and have a project you need coded,
             don't hesitate to contact me.</p>
         </div>
     </section>
+
+    <!-- Works section -->
+    <section id="projects">
+        
+        <h2 id="works" class="title">My Recent Works <hr></h2>      
+         
+                <img src="media/multipost.png" alt="Multipost image">
+        
+                <article class="grid-item" >
+                    <h2 class="title">Multi-Post Stories</h2>
+                    <p class="text">A daily selection of privately personalized reads; no accounts or sign-ups required. This has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</p>
+                    <ul class="btn no-style-deco">
+                       <li> <button type="button" class="grey-button">CSS</button></li>
+                       <li> <button type="button" class="grey-button">HTML</button></li>
+                        <li><button type="button" class="grey-button">Bootstrap</button></li>
+                       <li> <button type="button" class="grey-button">Ruby</button></li>
+                    </ul>
+                   <button type="button" class="orange-button">See project</button>
+                
+                </article>
+
+              <ul>
```

Comment:
- [ ] Good job with wrapping this work-section in a `section` tag as requested :clap: :clap: However, you seem to have forgotten the previous reviewer's suggestion [here](https://github.com/sylvester-wamaya/portfolio-website/pull/2#discussion_r1148393054) where he asked that you make this project card looks exactly like Figma's design. All elements on this card seem to aligned at the center instead of the left-alignment or flex-start seen in the Figma. Kindly implement this to fulfill the project requirement please :pray: